addons/account/models/account_move.py

```diff
@@ -4145,36 +4146,19 @@ def write(self, vals):
 
         if not self.env.context.get('tracking_disable', False):
             # Create the dict for the message post
-            tracking_values = {}  # Tracking values to write in the message post
+            tracking_value_ids_list = {}
             for move_id, modified_lines in move_initial_values.items():
-                tmp_move = {move_id: []}
                 for line in self.filtered(lambda l: l.move_id.id == move_id):
                     changes, tracking_value_ids = line._mail_track(ref_fields, modified_lines)  # Return a tuple like (changed field, ORM command)
-                    tmp = {'line_id': line.id}
                     if tracking_value_ids:
-                        selected_field = tracking_value_ids[0][2]  # Get the last element of the tuple in the list of ORM command. (changed, [(0, 0, THIS)])
-                        tmp.update({
-                            **{'field_name': selected_field.get('field_desc')},
-                            **self._get_formated_values(selected_field)
-                        })
-                    elif changes:
-                        field_name = line._fields[changes.pop()].string  # Get the field name
-                        tmp.update({
-                            'error': True,
-                            'field_error': field_name
-                        })
+                        tracking_value_ids_list[move_id] = tracking_value_ids
                     else:
                         continue
```

Comment:
```suggestion
                    tracking_value_ids_list[move_id] = line._mail_track(ref_fields, modified_lines)[1]
```

No need for the `changes` variable or for the `else: continue` anymore.
The `ìf tracking_value_ids:` forces you to do a useless `tracking_value_ids_list.get(move.id, [])` below

Reply:
This is related to your comment above. Do we want to continue logging changes if there are changes to untracked fields.

Reply:
You want to log changes only if there are changes done to tracked fields. Not sure to understand the question.

Reply:
`elif changes:` is a basic check for changes on untracked fields - and logs the generic message `[field] A modification has been operated on the line (id)`
I only get this condition satisfied when changing the tax tags

Reply:
My mistake... this is to track changes on many2many and one2many fields, as this is not available by default